src/linux/wifi/core/AccessPointControllerLinux.cxx

```diff
@@ -119,6 +119,94 @@ HostapdHwModeToPropertyValue(Wpa::HostapdHwMode hwMode)
         throw AccessPointControllerException(std::format("Invalid hostapd hw_mode value {}", magic_enum::enum_name(hwMode)));
     }
 }
+
+std::string
+IeeeAkmSuiteToWpaKeyMgmtPropertyValue(Ieee80211AkmSuite akmSuite)
+{
+    switch (akmSuite) {
+    case Ieee80211AkmSuite::Reserved0:
+        return "";
```

Comment:
Should probably remove this and let it go to the `default` case. Same with the others that are returning `""`, with the exception of `Ieee8021xSuiteB`.

Any value not appearing in hostapd's `hostapd_config_parse_key_mgm()` will be rejected. If this function was not hostapd-specific, I'd say leave it, but it'd be better to catch invalid input arguments to hostapd earlier.

Reply:
Updated for this function, but still need to do something similar for the other helpers. Need to think a bit more about those though